### PR TITLE
thomasgauvin: add new database examples to hyperdrive

### PR DESCRIPTION
**Pull request by thomasgauvin**

### CODE REVIEW

File: src/content/docs/hyperdrive/examples/pgedge.mdx [hunk 1]
```diff
@@ -6,20 +6,21 @@ title: Connect to pgEdge Cloud
...
+import { Render } from "~/components";
 
-This example shows you how to connect Hyperdrive to a [pgEdge](https://pgedge.com/) Postgres database. pgEdge Cloud provides easy deployment of fully-managed, fully-distributed, and secure Postgres. 
+This example shows you how to connect Hyperdrive to a [pgEdge](https://pgedge.com/) Postgres database. pgEdge Cloud provides easy deployment of fully-managed, fully-distributed, and secure Postgres.
 
 ## 1. Allow Hyperdrive access
 
-You can connect Hyperdrive to any existing pgEdge database with the default user and password 
+You can connect Hyperdrive to any existing pgEdge database with the default user and password
 provided by pgEdge.
```
Comment (Oxyjun) [inline src/content/docs/hyperdrive/examples/pgedge.mdx]:
```suggestion
You can connect Hyperdrive to any existing pgEdge database with the default user and password provided by pgEdge.
```

File: src/content/docs/hyperdrive/examples/azure.mdx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+### Azure Portal
+
+#### Public access networking
+
+To connect to your Azure Database for PostgreSQL instance using public Internet connectivity:
+
+1. In the [Azure Portal](https://portal.azure.com/), select the instance you want Hyperdrive to connect to.
+2. Expand **Settings** > **Networking** > ensure **Public access** is enabled > in **Firewall rules** add `0.0.0.0` as **Start IP address** and `255.255.255.255` as **End IP address**.
+3. Select **Save** to persist your changes.
+4. Select **Overview** from the sidebar and note down the **Server name** of your instance.
```
Comment (Oxyjun) [inline src/content/docs/hyperdrive/examples/azure.mdx:30]:
```suggestion
1. In the [Azure Portal](https://portal.azure.com/), select the instance you want Hyperdrive to connect to.
2. Expand **Settings** > **Networking**.
3. Go to **Firewall rules** and ensure 'Public access' is enabled.
4. Add `0.0.0.0` as 'Start IP address' and `255.255.255.255` as 'End IP address'.
5. Select **Save** to persist your changes.
6. Select **Overview** from the sidebar and note down the **Server name** of your instance.
```

File: src/content/docs/hyperdrive/examples/azure.mdx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+With the username, password, server name and database name (default: `postgres`), you can now create a Hyperdrive database configuration.
```
Comment (Oxyjun) [inline src/content/docs/hyperdrive/examples/azure.mdx]:
```suggestion
With the username, password, server name, and database name (default: `postgres`), you can now create a Hyperdrive database configuration.
```

**Review by Oxyjun — APPROVED** [review]

File: src/content/docs/hyperdrive/examples/azure.mdx (hunk 2 above)
Comment (thomasgauvin) [reply src/content/docs/hyperdrive/examples/azure.mdx:30]: I would recommend against 
```
3. Go to **Firewall rules** and ensure 'Public access' is enabled.
4. Add `0.0.0.0` as 'Start IP address' and `255.255.255.255` as 'End IP address'.
```

The IP ranges are in the Firewall rules section. Public access is its own section

File: src/content/docs/hyperdrive/examples/azure.mdx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+<Render file="public-connectivity" />
+
+### Azure Portal
+
+#### Public access networking
+
+To connect to your Azure Database for PostgreSQL instance using public Internet connectivity:
+
+1. In the [Azure Portal](https://portal.azure.com/), select the instance you want Hyperdrive to connect to.
+2. Expand **Settings** > **Networking** > ensure **Public access** is enabled > in **Firewall rules** add `0.0.0.0` as **Start IP address** and `255.255.255.255` as **End IP address**.
```
Comment (ReppCodes) [inline src/content/docs/hyperdrive/examples/azure.mdx:28]: I'm really looking forward to being able to link IP ranges here.

**Review by ReppCodes — COMMENTED** [review]
Looks good to me!

**Pull request merged**